src/components/LabelWrapper/index.tsx

```diff
@@ -27,7 +27,7 @@ export const LabelWrapper: FunctionComponent<LabelWrapperProps> = props => {
       {label ? (
         <Label {...otherProps}>
           <LabelText hasHintText={!!hintText}>
-            {label} {!required && <span>(optional)</span>}
+            {label} {required && <span>(required)</span>}
```

Comment:
we don't want required to appear in all the text input fields in the date input field